lib/WP_Auth0_LoginManager.php

```diff
@@ -144,12 +144,19 @@ public function login_auto() {
   }
 
   public function init_auth0() {
-    global $wp_query;
 
-    if ( $this->query_vars( 'auth0' ) === null ) {
+    // Not an Auth0 login process or settings are not configured to allow logins
+    if ( ! $this->query_vars( 'auth0' ) || ! WP_Auth0::ready() ) {
       return;
     }
 
+    // Catch any incoming errors and stop the login process
+    // See https://auth0.com/docs/libraries/error-messages
+    if ( $this->query_vars( 'error_description' ) ) {
+      $error_msg = sanitize_text_field( $this->query_vars( 'error_description' ) );
+      $this->die_on_login( $error_msg ? $error_msg : sanitize_text_field( $this->query_vars( 'error' ) ) );
```

Comment:
the `if` already checks if `error_description` is present on the query. Is there any case where the `sanitize_test_field(error_description)` will fail or give you something null/undefined? if not, it doesn't make sense to have this line's `if` that calls `sanitize_text_field(error)`

Reply:
We'll only ever get a scalar back from `query_vars()` and `sanitize_text_field()` is just there to sanitize. We're expecting error descriptions from the Auth0 server so if something comes through that's wonky then it's probably someone screwing around. If `sanitize_text_field()` returns an empty value then it was empty to begin with or invalid symbols:

https://developer.wordpress.org/reference/functions/sanitize_text_field/